partner_firstname/__init__.py

```diff
@@ -15,8 +17,5 @@
 #
 #    You should have received a copy of the GNU Affero General Public License
 #    along with this program.  If not, see <http://www.gnu.org/licenses/>.
-#
-##############################################################################
 
-from . import partner
-from . import res_user
+from . import models
```

Comment:
* Making this it seems you used `git rm/git add` commands. Would not be better to use `git mv` to avoid to lose the file history?;
* in addition it would be better to rename the file `partner.py` to `models/res_partner.py`